onlineShop.core/Products/Book.cs

```diff
@@ -8,14 +8,18 @@ public class Book: PhysicalProduct
 
         public string Author { get; set; }
 
+        public string Genre { get; set; }
```

Comment:
better use tags to put the genre. A book can have multiple genres. e.g. scifi, romance